hyperon_das/das.py

```diff
@@ -20,6 +20,29 @@
 
 class DistributedAtomSpace:
     def __init__(self, system_parameters: Dict[str, Any] = {}, **kwargs) -> None:
+        """Creates a new DAS object
+
+        Args:
+            system_parameters (Dict[str, Any], optional): Sets the system parameters. 
+                Defaults to {'running_on_server': False, 'cache_enabled': False, 'attention_broker_hostname': 'localhost', 'attention_broker_port': 27000}.
+
+        Keyword Args:
+            atomdb (str, optional): AtomDB type suported values are 'ram' and 'redis_mongo'. Defaults to 'ram'
+            query_engine (str, optional): Set the type of connection for the query engine, values are 'remote' or 'local'. Defaults to 'local'
+            host (str, optional): Sets the host for the remote query engine, it's mandatory when the query_engine is equal to 'remote'.
+            port (str, optional): Sets the port for the remote query engine, it's mandatory when the query_engine is equal to 'remote'.
+            mongo_hostname (str, optional): MongoDB hostname. Defailts to 'localhost'
```

Comment:
Typo: Defailts --> Defaults. Please check a lot of occurrences of the same typo.